package.json

```diff
@@ -15,10 +15,7 @@
     "prepare": "husky install"
   },
   "lint-staged": {
-    "*.@(ts|tsx)": [
-      "yarn lint",
-      "yarn format"
-    ]
+    "*.(ts|tsx)": "eslint --cache --fix"
```

Comment:
I'm very down to do `yarn format` since windows downloads funny and causes annoying `LF` issues, but in that case, we should make `pre-commit` run `yarn pre-commit` rather than `lint-staged` since definitions would be misleading/wrong in this `lint-staged` case

Reply:
Sorry, I'm a bit confused - I don't see a `yarn pre-commit` script?

I think `yarn format` by itself is probably good for the `lint-staged` action, since it is a superset of `yarn lint` as of now.

What are your thoughts?

Reply:
Yea like adding in a `pre-commit` script and just not using `lint-staged`? I’m down for that if that sounds good to you?

Reply:
I think I’ll just make the change and you can lmk if you like it

Reply:
Oh I see, like [this pre-commit](https://pre-commit.com/)? Thought you meant the pre-commit part of `lint-staged`. I don't mind, with the caveat that we'd need to make our other developers use this tool *as well*, since it's not part of the npm ecosystem. I think that's a core advantage of using `lint-staged`, in that it gets automatically installed as part of `yarn`.

What do you think? If `pre-commit` is that much better, then we can switch to that.

Reply:
not that, much simpler, I just made the change

thoughts on PR now? feel free to merge if it looks good

Reply:
Ah, you just meant adding a new script line to `package.json` - think I misunderstood earlier. LGTM!